e2e/cypress/integration/team_settings/create_a_team_spec.js

```diff
@@ -52,4 +46,56 @@ describe('Teams Suite', () => {
         // * Team name should displays correctly at top of LHS
         cy.get('#headerTeamName').should('contain', 'Team Test');
     });
+
+    it.only('MM-T1437 Try to create a new team using restricted words', () => {
+        // * Enter different reserved words and verify the error message
+        tryReservedTeamURLAndVerifyError('plugins');
+        tryReservedTeamURLAndVerifyError('login');
+        tryReservedTeamURLAndVerifyError('admin');
+        tryReservedTeamURLAndVerifyError('channel');
+        tryReservedTeamURLAndVerifyError('post');
+        tryReservedTeamURLAndVerifyError('api');
+        tryReservedTeamURLAndVerifyError('oauth');
+        tryReservedTeamURLAndVerifyError('error');
+        tryReservedTeamURLAndVerifyError('help');
+    });
 });
+
+function openTeamCreationModalFromHamburgerMenu() {
+    // # Click hamburger main menu
+    cy.get('#sidebarHeaderDropdownButton').click();
+
+    // * Dropdown menu should be visible
+    cy.get('#sidebarDropdownMenu').should('exist').within(() => {
+        // # Click "Create a Team"
+        cy.findByText('Create a Team').should('be.visible').click();
+    });
+}
+
+function tryReservedTeamURLAndVerifyError(teamURL) {
+    // # Open the create team fullscreen modal from hamburger menu
+    openTeamCreationModalFromHamburgerMenu();
+
+    // # Input a random team name
+    cy.get('#teamNameInput').should('be.visible').type('team-name-123');
```

Comment:
I'm good as is but passing `teamURL` or team name will work too.
```suggestion
    cy.get('#teamNameInput').should('be.visible').type(teamURL);
```